orange/src/main/java/com/hartwig/hmftools/orange/OrangeWGSRefConfig.java

```diff
@@ -76,17 +75,14 @@ static void registerConfig(@NotNull ConfigBuilder configBuilder)
     @NotNull
     String chordPredictionTxt();
 
-    @NotNull
-    String cuppaResultCsv();
-
-    @NotNull
-    String cuppaSummaryPlot();
+    @Nullable
+    String cuppa2VisData();
 
     @Nullable
-    String cuppaFeaturePlot();
+    String cuppa2VisPlot();
 
     @Nullable
-    String cuppaChartPlot();
+    String cuppa2PredSumm();
```

Comment:
Prediction summary? Also I didn't notice where this was used, other than being populated.

Also, these three cuppa2 entries are all paths, e.g. elsewhere you used `String cuppaVisDataTsv = config.wgsRefConfig().cuppa2VisData();` so maybe it makes sense to just name the field with the Tsv suffix as well? We also seem to use the suffixes File, Directory where appropriate. But plot files are just called ...Plot for whatever reason, so that one still works 😄.

Reply:
Also, I believe @kduyvesteyn was interested in a way for the cuppa run mode to be passed through to orange. Didn't notice that that was possible yet, can it be unambiguously determined from the cuppa2 data that is loaded already, or do we need an additional param to be passed in by the caller of orange for this to be possible?

Reply:
I've now removed the unused`cuppa2PredSumm()` property